mealpy.py

```diff
@@ -136,4 +161,4 @@ def execute_reserve_meal():
 
 
 if __name__ == '__main__':
-    execute_reserve_meal()
+    cli()
```

Comment:
Could this also be `cli_group()`? This might already be covered in `click` docs but I haven't gotten around to reading in depth yet.

Reply:
Yes, but I have just added a new group so we need to use the collection.

Reply:
I couldn't use the `CommandCollection` with multiple groups for some reason, so I switched to using a decorator to load the `MealPal` object into the relevant commands that require them.